src/main/webapp/resources/js/pages/user/components/UserAccountLayout.jsx

```diff
@@ -1,33 +1,66 @@
 import React from "react";
-import { Outlet } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { Outlet, useParams } from "react-router-dom";
 import { Col, Row } from "antd";
 import { PageWrapper } from "../../../components/page/PageWrapper";
 import UserAccountNav from "./UserAccountNav";
 import { setBaseUrl } from "../../../utilities/url-utilities";
-import { isAdmin } from "../../../utilities/role-utilities";
+import { useGetUserDetailsQuery } from "../../../apis/users/users";
+import { ContentLoading } from "../../../components/loader";
+import { setUserDetails } from "../services/userReducer";
 
 /**
  * React component that layouts the user account page.
  * @returns {*}
  * @constructor
  */
 export default function UserAccountLayout() {
+  const dispatch = useDispatch();
+  const { userId } = useParams();
+  const {
+    data: userDetails,
+    isLoading,
+    isSuccess,
+  } = useGetUserDetailsQuery(userId);
+
   const goToAdminUserListPage = () =>
     (window.location.href = setBaseUrl(`admin/users`));
 
+  if (isSuccess) {
```

Comment:
Change this to use `React.useEffect`.  Set `isSuccess` to a dependency

Reply:
Oops. Please see 4b40280a777ff6d4bed8d0755b57064c8b4fab2c.